src/tests/ftest/util/server_utils_params.py

```diff
@@ -319,15 +319,31 @@ def __init__(self, index=None):
             self.pinned_numa_node = BasicParameter(None)
             self.log_mask = BasicParameter(None, "DEBUG,RPC=ERR")
             self.log_file = LogParameter(log_dir, None, "daos_server.log")
-            self.env_vars = BasicParameter(
-                None,
-                ["ABT_ENV_MAX_NUM_XSTREAMS=100",
-                 "ABT_MAX_NUM_XSTREAMS=100",
-                 "DAOS_MD_CAP=1024",
-                 "FI_SOCKETS_MAX_CONN_RETRY=1",
-                 "FI_SOCKETS_CONN_TIMEOUT=2000",
-                 "DD_MASK=mgmt,io,md,epc,rebuild"]
-            )
+
+            # Set extra envariables for sockets provider
+            if self.provider is "ofi+sockets":
```

Comment:
(pylint-no-member) Instance of 'PerServerYamlParameters' has no 'provider' member